unit_testing/unit_test.py

```diff
@@ -14,7 +51,15 @@
 import xarray as xr
 import matplotlib.pyplot as plt
 import datetime
+import os.path as path
 import logging
+from . import general_utils
```

Comment:
This breaks if you are not in ``COAsT/coast``. Is there a way to use ``os.path`` to sort this? Not sure. Not a big issue.

Reply:
Maybe putting it into __init__.py like the object files. I'll look into it